staging/src/k8s.io/kubectl/pkg/cmd/apiresources/apiresources_test.go

```diff
@@ -211,6 +212,77 @@ bazzes.somegroup
 `,
 			expectedInvalidations: 1,
 		},
+		{
+			name: "output json",
+			commandSetupFn: func(cmd *cobra.Command) {
+				cmd.Flags().Set("output", "json")
+			},
+			expectedOutput: `{
+	"kind": "v1",
```

Comment:
Check your indentation on the `expectedOutput`.  I think it might need to be spaces instead of tabs in order to match the output, so this test case will pass.

Reply:
That did the trick. Thanks 👍🏽 
What is the purpose of `expectedInvalidations` field there? I couldn't find anything.